source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -243,6 +242,37 @@ private void stopWaveTimer() {
     }
   }
 
+  /**
```

Comment:
Commented out code block should be removed - was a code smell fix earlier today

Reply:
got it